gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ExternalTableType.java

```diff
@@ -298,7 +305,7 @@ public boolean equals(Object obj) {
    *     of 10 GB maximum size across all URIs.
    * @param schema the schema for the external data
    * @param format the source format of the external data
-   * @return a builder for an ExternalDataConfiguration object given source URIs, schema and format
+   * @return a builder for an ExternalTableDefinition object given source URIs, schema and format
```

Comment:
Should this be ExternalTableType instead of ExternalTableDefinition (here and below)?